llvm/test/Transforms/LoopVectorize/AArch64/sve-structured-store.ll

```diff
@@ -0,0 +1,279 @@
+; RUN: opt -mtriple=aarch64 -mattr=+sve -S -passes=loop-vectorize,interleaved-access  -enable-masked-interleaved-mem-accesses=true -sve-gather-overhead=10 -sve-scatter-overhead=10  -enable-shuffle-padding=true -force-vector-width=16 -aarch64-sve-vector-bits-min=512 < %s -o - | FileCheck %s --check-prefixes=ENABLE-SHUFFLE-PADDING
```

Comment:
The `-sve-gather-overhead=10 -sve-scatter-overhead=10` flags aren't needed because they are the default already I think. Also, the `-aarch64-sve-vector-bits-min=512` flag contradicts the `vscale_range(2,2)` attribute (i.e. min=256 bits).

Reply:
Fixed, remove unnecessary tokens.